google-cloud-tools-plugin/src/com/google/gct/idea/appengine/util/CloudSdkUtil.java

```diff
@@ -117,6 +117,10 @@ public static String toSdkHomeDirectory(String sdkExecutablePath) {
     return null;
   }
 
+  public static String getSystemCommand() {
```

Comment:
No to dwell on this too much, but it was a pretty weird test to write (given that there's no ready way to mock SystemInfo and I'm not sure that's worth the effort). It basically just duplicates the code in the unit test:

 ``` 
 @Test
  public void testSystemDependentCommand() {
    String systemCommand = SystemInfo.isWindows ? WIN_COMMAND : UNIX_COMMAND;
    assertEquals(systemCommand, CloudSdkUtil.getSystemCommand());
  }
```